va/validation-authority.go

```diff
@@ -117,9 +117,14 @@ func (va ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdentif
 		// connection immediately.
 		DisableKeepAlives: true,
 	}
+	logRedirect := func(req *http.Request, via []*http.Request) error {
+		va.log.Notice(fmt.Sprintf("validateSimpleHTTP [%s] redirect from %q to %q", identifier, via[len(via)-1].URL.String(), req.URL.String()))
```

Comment:
Question fo rthe team: is this what should be going in the audit logs? That seems a little too much. I thought it was just for data relevant to determining bad requests.

Which actually brings up another thing which is: should we be using some kind of structured format for the various places it logs, so we can use that data later?

Perhaps, I've misunderstood the purpose of the audit logger.